.github/workflows/diracinstall.yml

```diff
@@ -0,0 +1,31 @@
+name: dirac-install
+
+on: [push, pull_request]
+
+jobs:
+  diracInstall:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: False
+      matrix:
+        python:
+          - 2.6.9
+          - 2.7.5
+          - 2.7.13
+          - 3.6.8
+
+    steps:
+    - uses: actions/checkout@v2
+      # with:
+      #  ref: pcompatibility
+    - name: prepare environment
+      run: |
+        conda config --set add_pip_as_python_dependency false
+        conda create -c conda-forge -c free -n python_${{ matrix.python }} python=${{ matrix.python }}
+        # curl -O https://raw.githubusercontent.com/DIRACGrid/DIRAC/integration/Core/scripts/dirac-install.py
+    - name: run dirac-install script
+      run: |
+        eval "$(conda shell.bash hook)" && conda activate python_${{ matrix.python }}
+        # python dirac-install.py -l DIRAC -r integration -t server  --dirac-os -m https://github.com/zmathe/DIRAC.git:::DIRAC:::pcompatibility -ddd
+        python Core/scripts/dirac-install.py -l DIRAC -r integration -t server  --dirac-os -m https://github.com/zmathe/DIRAC.git:::DIRAC:::pcompatibility -ddd
```

Comment:
I guess this version is just for your tests

Reply:
BTW I have added "master" in https://github.com/DIRACGrid/DIRAC/pull/4377 which is probably what we want to use here.